internal/webapi/server.go

```diff
@@ -47,6 +48,7 @@ func NewServer(db database.Database, auth authentication.Authenticator[APIAuthCr
 
 	femto.OnGet(mux, "/api/stats/all", api.AllStatistics)
 	femto.OnGet(mux, "/api/stats/offline", api.HandleOfflineMachineRequest)
+	femto.OnGet(mux, "/api/stats/deltas", api.durationDelta)
```

Comment:
NIT: I'm not sure this is a great name for this endpoint, as it's not super descriptive. How about something like `/api/stats/since_last_seen`?